misc/zpipe_append.c

```diff
@@ -210,7 +213,7 @@ static int inf(FILE *source, FILE *dest, int window_bits, int _flush,
 		return Z_ERRNO;
 
 	out = malloc(CHUNK_o);
-	if (in == NULL) {
+	if (out == NULL) {
```

Comment:
Yes, it did generate them correctly for me. Would you be able to confirm, please?